integration/app.py

```diff
@@ -59,81 +66,20 @@ def get_logger_data(exception):
     def list_cards():
         try:
             response_data = shopper_payments_adapter.list_cards()
```

Comment:
One thing about this, how are we expecting the response in this method? I was checking the implementation for bgc I saw that some methods are returning Dict instead of Response, in this case, it seems to be a Dict. If this is the case we should use Response since in that way we can ensure that the local implementation is returning the expected value, if we don't push this it could be that the integrators return a different format and we just return that format to the backend.
It is not necessary to check this in this PR but if this is the case we have to apply the change here and just return to a Response type in the implementation

```suggestion
            response_data = shopper_payments_adapter.list_cards()
            return jsonify(response_data)
```